static/dart/lib/component/profile/posts.html

```diff
@@ -12,10 +12,83 @@
   <img ng-show='loading > 0' src='/static/img/spinner-blue-on-white.gif'>
 </h1>
 
-<p>
-  Showing {{pager.startingAt | number}}-{{pager.endingAt | number}}
-  of {{pager.totalResults | number}} posts.
-</p>
+<div class='row'>
+  <div class="col-sm-5">
+    Showing {{pager.startingAt | number}}-{{pager.endingAt | number}}
+    of {{pager.totalResults | number}} posts.
+  </div>
+  <div class='col-sm-7'>
+    <div ng-show="failedTasks">
+      <div class="alert alert-danger" role="alert">
+        <div class="row">
+          <div class="col-sm-8">
+            <em>
+               Errors were reported by workers for this profile. View  and delete them 
+               to remove this message.
+                
+                
+            </em>
+          </div>
+          <div class="col-sm-4">
+            <span class="pull-right">
+              <a 
+                class="btn btn-default" 
+                href="/admin/background-tasks" 
+                ng-show="auth.isAdmin()">
+                View
+              </a>
+              <a 
+                class="btn btn-default" 
+                ng-show="auth.isAdmin() == false"
+                href="/admin/background-tasks" 
+                disabled>
+                You must be an admin to manage background tasks 
```

Comment:
This seems like it might be confusing to an ordinary user, who is powerless to remove this error message and probably doesn't know what background tasks or workers are.